packages/documentation/src/stories/getting-started/components/components.docs.mdx

```diff
@@ -12,26 +13,35 @@ import SampleJsSpecific from './components-js-specific.sample?raw';
 
 [![npm version](https://badge.fury.io/js/@swisspost%2Fdesign-system-components.svg)](https://badge.fury.io/js/@swisspost%2Fdesign-system-components)
 
-A set of standard web components for easy integration with every framework or no framework at all.
+A set of standard web-components for easy integration with every framework or no framework at all.<br/>
```

Comment:
```suggestion
A set of standard Web Components for easy integration with every framework or no framework at all.<br/>
```
It's correct without dash: https://developer.mozilla.org/en-US/docs/Web/API/Web_Components. Maybe we can capitalize the words instead.